.travis.yml

```diff
@@ -11,11 +11,11 @@ sudo: false
 env:
   global:
     - PYENV_VERSION=3.6
-    - CHANS_DEV="-c pyviz/label/dev -c jbednar"
+    - CHANS_DEV="-c pyviz/label/dev -c jbednar -c bokeh/label/dev"
```

Comment:
Just to check: is this addition a build time channel requirement, or do users need it too?

Reply:
Build time dependency only